src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -355,6 +362,25 @@ internal class RecommendationService(
     }
   }
 
+  private fun sendRecommendationStartedEvent(recommendationId: Long) {
+    val crn = recommendationRepository.findById(recommendationId).map { it.data.crn }.get()
+    val payload = MrdEvent(
+      message = MrdEventMessageBody(
+        eventType = "prison-recall.recommendation.started",
+        version = 1,
+        description = "Recommendation started (recall or no recall)",
+        occurredAt = LocalDateTime.now(),
+        detailUrl = "", // TODO TBD
+        personReference = PersonReference(listOf(TypeValue(type = "CRN", value = crn))),
+        additionalInformation = AdditionalInformation(recommendationUrl = "http://mrd.case.crn/overview")
+      ),
+      signingCertURL = "https://sns.us-east-1.amazonaws.com/SimpleNotificationService-0000000000000000000000.pem",
```

Comment:
I don't think you need to add this, when I spoke to Matt about it the other day he said: "The rest of the wrapper attributes will be added by SNS automatically including the subscribeUrl and signingCertURL fields". 
So you should be good to remove these 2 fields.